ImageInLib/ImageInLib/src/imageInterpolation.c

```diff
@@ -1,3 +1,4 @@
+#include"../src/data_storage.h"
```

Comment:
is this relative path necesary?
Is not data_storage.h enough?

Reply:
yes, because if not I can not find the needed file